### DISCUSSION TITLE
Upgrade PostgreSQL on Docker Compose

### DISCUSSION THREAD

**Discussion by SionS4R** [goauthentik/authentik#10298] [General]
Hello everyone,
unfortunately I am stuck with the PostgreSQL upgrade, it keeps failing and I go back to a fresh installation with everything deleted.

Can anyone help me in a step by step guide, because if I follow this guide perfectly I go into error?
https://docs.goauthentik.io/docs/troubleshooting/postgres/upgrade_docker

Thank you so much for your support in upgrading to the latest version

**Comment by AblabiX** [comment]
same problem at step _Apply your backup to the new database_

cant find the user and role on the new 16 version db
